packages/styles/scss/patterns/blocks/content-group-horizontal/_content-group-horizontal.scss

```diff
@@ -7,3 +7,15 @@
 
 @import '../../../internal/content-block/content-block';
 @import '../../../components/content-item-horizontal/content-item-horizontal';
+
+@mixin content-group-horizontal {
+  .#{$prefix}--content-group-horizontal {
+    &--border {
+      border-bottom: solid 1px $ui-04;
```

Comment:
@greeshmareddyg I don't think this is what we were talking about for this. We were thinking of setting the border to true inside of `ContentBlock`:

```
<ContentBlock 
  heading={heading} 
  aside={{
    items: <div></div>
    border: true
  }}
>
    {items.map((item, index) => (
        <ContentItemHorizontal ... />
      ))}
</ContentBlock>
```

Reply:
@jeffchew addressed this, still gave it as optional with default border true, is it ok.